core/client/verifier/verifier.go

```diff
@@ -53,6 +52,8 @@ type LogTracker interface {
 	VerifyLogRoot(state *pb.LogRootRequest, newRoot *pb.LogRoot) (*types.LogRootV1, error)
 }
 
+type LogTrackerFactory func(*tclient.LogVerifier) LogTracker
```

Comment:
Given the length of the conversation we had around this yesterday, I think a short description here would be worthwhile. It only needs to say that this allows different implementation strategies, including a no-op implementation, for a Tracker to be injected based on a verifier. It feels obvious now, but I'm not sure that would be true without the context.